### PR TITLE
[console-wallet] Add MinedUnconfirmed to command mode wait stage

### PR DESCRIPTION
**Pull request by delta1**
## Description

Adds `MinedUnconfirmed` to the command mode wait stage, when running commands makes it possible to wait until transactions are mined but not yet confirmed, since `Mined` will wait for the configured number of confirmations. Bumped the default wait stage timeout from 300 seconds to 600. 

Also adds a readme with some documentation regarding the available commands in command mode.

## Motivation and Context

MinedUnconfirmed was missing from the command mode wait stage enum 

## How Has This Been Tested?

Manually 

## Types of changes

* [ ] Bug fix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [x] Feature refactor (No new feature or functional changes, but performance or technical debt improvements)
* [ ] New Tests
* [x] Documentation

## Checklist:

* [x] I'm merging against the `development` branch.
* [x] I ran `cargo-fmt --all` before pushing.
* [ ] I ran `cargo test` successfully before submitting my PR.
* [x] I have squashed my commits into a single commit.
* [ ] My change requires a change to the documentation.
* [x] I have updated the documentation accordingly.
* [ ] I have added tests to cover my changes.

### CODE REVIEW

File: applications/tari_console_wallet/README.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+1. send-tari 1.000000 T c69fbe5f05a304eaec65d5f234a6aa258a90b8bb5b9ceffea779653667ef2108 coffee
+Monitoring 1 sent transactions to Broadcast stage...
+Done! All transactions monitored to Broadcast stage.
+```
+
+- **make-it-rain**
+
+Make it rain! Send many transactions to a public key or emoji id.
+
+`tari_console_wallet --command "make-it-rain <tx/sec> <duration> <amount> <increment> <start time or now> <pubkey> <message>"`
```
Comment (SWvheerden) [inline applications/tari_console_wallet/README.md]:
```suggestion
`tari_console_wallet --command "make-it-rain <tx/sec> <duration> <amount> <increment> <start time or now> <pubkey> <optional message>"`
```

Comment (SWvheerden) [reply applications/tari_console_wallet/README.md]: Message field is not required, this hints that it is.

File: applications/tari_console_wallet/README.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+Done! All transactions monitored to Broadcast stage.
+```
+
+- **coin-split**
+
+Split one or more unspent transaction outputs into many.
+Creates a transaction that must be mined before the new outputs can be spent.
+
+`tari_console_wallet --command "coin-split <amount per coin> <number of coins>"`
+
```
Comment (SWvheerden) [inline applications/tari_console_wallet/README.md]:
```suggestion
example:
$ tari_console_wallet --command "coin-split 10000 9"
```

**Review by SWvheerden — DISMISSED** [review]
LGTM
I think we can improve the readme, left comments there.

File: applications/tari_console_wallet/README.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+Available balance: 1268922.299856 T
+Pending incoming balance: 6010 µT
+Pending outgoing balance: 1.337750 T
+```
+
+- **send-tari**
+
+Send an amount of Tari to a public key or emoji id.
+
+`tari_console_wallet --command "send-tari <amount> <pubkey> <message>"`
```
Comment (delta1) [inline applications/tari_console_wallet/README.md]:
```suggestion
`tari_console_wallet --command "send-tari <amount> <pubkey> <optional message>"`
```

**Review by StriderDM — APPROVED** [review]
LGTM

**Pull request merged**